internal/store/migrations.go

```diff
@@ -84,58 +84,6 @@ var migrations = []migration{
 			if err != nil {
 				return err
 			}
-		} else if e.DriverName() == driverSqlite {
```

Comment:
I believe we should leave the migrations as they were for now even though it's highly unlikely it will matter. My thinking here is if we are reviewing master code to look for an issue that may have been introduced in migrations that we should have the full history even for SQLite just in case.

It looks like we are keeping the `driverSqlite` variable for now so that should be okay.

Same for the others below. If you ran into something that led to wanting to remove these just let me know and we can discuss.